libs/ledger/src/lib/ledger-manager.tsx

```diff
@@ -47,9 +52,20 @@ export const LedgerManager = ({ partyId }: { partyId: string }) => {
   useEffect(() => {
     setDataCount(gridRef.current?.api?.getModel().getRowCount() ?? 0);
   }, [extractedData]);
+  const VEGA_URL = useEnvironment((store) => store.VEGA_URL);
+  const protohost = VEGA_URL ? getProtoHost(VEGA_URL) : '';
 
   return (
     <div className="h-full relative">
+      {protohost && (
+        <Link
+          className="h-[30px] text-sm p-2"
+          title={t('Download all to .csv file')}
+          href={`${protohost}/api/v2/ledgerentry/export?partyId=${partyId}`}
```

Comment:
you will require both a partyId and an assetId.

Reply:
assetID selector has been added